data/seeds/01-users.js

```diff
@@ -0,0 +1,9 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users')
+    .truncate()
+    .then(function() {
+      // Inserts seed entries
+      return knex('users')
```

Comment:
I don't believe this'll really seed the database, since the table gets truncated and then you return what's now in it (nothing; an empty array). If your intended purpose is to simply truncate the database, then forego the extra step:
```javascript
exports.seed = function(knex, Promise) {
  return knex('users')
    .truncate()
}
```